upgrades/upgrade_testing_framework/steps/step_post_upgrade_robot_test.py

```diff
@@ -0,0 +1,22 @@
+from upgrade_testing_framework.core.framework_step import FrameworkStep
+from robot_tests.test_executor import TestExecutor
+
+class PostUpgradeRobotTest(FrameworkStep):
+    """
+    This step is where you run post-upgrade tests
+    """
+
+    def _run(self):
+        # Get the state we need
+        target_cluster = self.state.target_cluster
+
+        # Pull host and credentials (if security is enabled) from state
+        test_executor = TestExecutor("localhost", target_cluster.rest_ports[0])
+
+        # Begin the step body
+        test_executor.execute_tests(include_tags=self.state.eligible_expectations,
```

Comment:
This one is pretty important to me - please follow the FrameworkStep convention of always pulling state into a local variable in the dedicated section at the beginning of the `_run()` definition.  As FrameworkSteps get more complex, this will keep us from pulling out hair out.  The state is basically a global variable and we should treat it with a combination of reverence/revulsion and keep it in a very explicit box.

Reply:
Similarly, please never write to the global state outside the prescribed section at the end of a FrameworkStep

Reply:
Good thought will add in